hudi-common/src/main/java/org/apache/hudi/metadata/HoodieTableMetadata.java

```diff
@@ -112,6 +113,14 @@ static boolean isMetadataTable(String basePath) {
     return basePath.endsWith(HoodieTableMetaClient.METADATA_TABLE_FOLDER_PATH);
   }
 
+  static boolean isMetadataTableSecondaryIndexPartition(String basePath, Option<String> partitionName) {
+    if (!isMetadataTable(basePath) || !partitionName.isPresent()) {
```

Comment:
Maybe we can move this utility to `HoodieTableMetadataUtil`.

Reply:
removed.. this method is not required after i added `MetadataPartitionType#fromPartitionPath`